.changeset/three-points-collect.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I think we need a comment explaining why the base branch is being changed to origin/dev

Reply:
But it is about our internal development process. We don't publish it to `npm`.